com.github.paolostivanin.OTPClient.json

```diff
@@ -0,0 +1,82 @@
+{
+  "app-id": "com.github.paolostivanin.OTPClient",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "3.26",
+  "sdk": "org.gnome.Sdk",
+  "command": "otpclient",
+  "finish-args": [
+    "--filesystem=~/.config",
+    "--share=ipc",
+    "--socket=session-bus",
```

Comment:
What do you need session bus access for? Best practice is to run your app with the --log-session-bus switch to flatpak and then whitelist the places you need to talk to.

Reply:
Yesterday it wasn't working without that line... (error: `gdbus.Error:org.freedesktop.DBus.Error.ServiceUnknown`).

Now I have deleted it, rebuilt the app, and it's working xD

Any idea why?